From d993f376a02ac241cb9c9dc51534def1ab78ece2 Mon Sep 17 00:00:00 2001
From: Adam Ford <aford173@gmail.com>
Date: Tue, 6 Oct 2020 12:20:55 -0500
Subject: [PATCH 62/62] arm64: dts: beacon-imx8mn-hdmi: Enable IRQ

With the new baseboards, the interrupts can function better on Nano.
This patch enables the IRQ on the HDMI bridge to operate like it
does on the Mini.

Fixes: PHOENIX8-198

Signed-off-by: Adam Ford <aford173@gmail.com>
---
 arch/arm64/boot/dts/freescale/beacon-imx8mn-hdmi.dts | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/arch/arm64/boot/dts/freescale/beacon-imx8mn-hdmi.dts b/arch/arm64/boot/dts/freescale/beacon-imx8mn-hdmi.dts
index a60c71c5d778..c391b10f285e 100644
--- a/arch/arm64/boot/dts/freescale/beacon-imx8mn-hdmi.dts
+++ b/arch/arm64/boot/dts/freescale/beacon-imx8mn-hdmi.dts
@@ -73,6 +73,8 @@
 		adi,dsi-lanes = <4>;
 		pinctrl-names = "default";
 		pinctrl-0 = <&pinctrl_synaptics_dsx_io>;
+		interrupt-parent = <&gpio1>;
+		interrupts = <9 IRQ_TYPE_LEVEL_LOW>;
 		#sound-dai-cells = <0>;
 		status = "okay";
 
-- 
2.17.1

